src/gax/Cargo.toml

```diff
@@ -13,46 +13,46 @@
 # limitations under the License.
 
 [package]
-name                 = "gcp-sdk-gax"
-version              = "0.2.0"
-description          = "Google Cloud Client Libraries for Rust"
+authors.workspace    = true
```

Comment:
nit: I think it is less distracting to keep the `*.workspace` entries at the end 🤷

Reply:
I will double check but I did not intentionally change this. I wonder if it was my editor or CLI...

Reply:
restored, I think my editors built in formatted like things alphabetical.